client/blocks/image-editor/index.jsx

```diff
@@ -41,6 +47,22 @@ const ImageEditor = React.createClass( {
 		onCancel: PropTypes.func,
 		onReset: PropTypes.func,
 		className: PropTypes.string,
+		defaultAspectRatio: PropTypes.oneOf( [
+			AspectRatios.FREE,
+			AspectRatios.ORIGINAL,
+			AspectRatios.ASPECT_1X1,
+			AspectRatios.ASPECT_16X9,
+			AspectRatios.ASPECT_4X3,
+			AspectRatios.ASPECT_3X2
+		] ),
+		allowedAspectRatios: PropTypes.arrayOf( PropTypes.oneOf( [
+			AspectRatios.FREE,
+			AspectRatios.ORIGINAL,
+			AspectRatios.ASPECT_1X1,
+			AspectRatios.ASPECT_16X9,
+			AspectRatios.ASPECT_4X3,
+			AspectRatios.ASPECT_3X2
+		] ) ),
```

Comment:
Minor, but we can use a constant for the array if you think these two won't diverge.